typings/index.d.ts

```diff
@@ -557,14 +557,15 @@ declare module 'discord.js' {
       DEFAULT: 0x000000;
       WHITE: 0xffffff;
```

Comment:
Add Discord's black?
```suggestion
      WHITE: 0xffffff;
      BLACK: 0x23272a;
```

Reply:
The black color didn't change I think because it's the same as `NOT_QUITE_BLACK`

Reply:
Ah, I see. Didn't notice that 😅 

Maybe it should be changed to `BLACK` so it's a more straightforward 🤔 just a thought though

Reply:
I think we can keep as is for now and see what the maintainers think 😄

Reply:
The new BLACK on the branding guidelines is #000000.

The old "black" can stay as it is